content/services/u-s-digital-registry.md

```diff
@@ -8,12 +8,23 @@ aliases:
   - /services/the-federal-mobile-apps-registry/
 ---
 
-Whether for access to emergency, financial or education public services, users need to trust they are engaging with official U.S. government digital accounts.
 
-To help prevent exploitation from unofficial sources, phishing scams, or malicious entities, the [U.S. Digital Registry](https://usdigitalregistry.digitalgov.gov/) serves as a resource for agencies, citizens, and developers to confirm the official status of social media and public-facing collaboration accounts, mobile apps, and mobile websites. The accounts in the registry are independently updated by federal managers across the government who maintain individual agency accounts.
+To help prevent exploitation from unofficial sources, phishing scams, or malicious entities, the [**U.S. Digital Registry**](https://usdigitalregistry.digitalgov.gov/) serves as a resource for agencies, citizens, and developers to confirm the official status of social media and public-facing collaboration accounts, mobile apps, and mobile websites.
 
-### How to Access the U.S. Digital Registry
+The accounts in the registry are independently updated by federal managers across the government who maintain individual agency accounts.
+
+**Notice:** As of April 19, 2018, all entries that have not been updated by agencies since January 1, 2017 have been archived. Archived accounts will not appear in the public facing list and API of official, active accounts. This was done to help ensure that users can trust that accounts listed in the U.S. Digital Registry are official and active.
+
+If your federal agency account has been archived but is still active, you can easily log-in and update these accounts to make them active again. In the meantime, GSA will continue to help agencies fulfill this need by providing support and guidance.
```

Comment:
`you can easily log-in and update these accounts to make them active again` could be something like `you should log in to the [U.S. Digital Registry](https://usdigitalregistry.digitalgov.gov/) directly to update your account and make it active again.`